internal/db/queryBuilder.go

```diff
@@ -149,18 +163,18 @@ func (qb *queryBuilder) queryWithRemovals() string {
 	`
 
 	selectStatement := `
-		SELECT path, mode, size, bytes, packed, deleted, (hash).h1, (hash).h2
+		SELECT path, mode, size, (cache_hash).h1 cache_h1, (cache_hash).h2 cache_h2, bytes, packed, deleted, (hash).h1, (hash).h2
```

Comment:
Is the `cache_hash` different than the `hash`?

It seems like we may be returning the same values twice. Could we always return `hash` and then include a `is_hashed` boolean?

Reply:
Yeah possibly. I don't know why `hash` isnt't always returned; it was added during one of the rebases. I'll try to find out.